sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -311,6 +376,108 @@ def __repr__(self):
         )
 
 
+class HealthcareEntity(DictMixin):
+    """HealthcareEntity contains information about a Healthcare entity found in text.
+
+        :ivar str text: Entity text as appears in the request.
+        :ivar str category: Entity category, such as Dosage or MedicationName, etc.
+        :ivar str subcategory: Entity subcategory.  # TODO: add subcategory examples
+        :ivar int offset: The Healthcare entity text offset from the start of the document.
+        :ivar int length: The length of the Healthcare entity text.
+        :ivar float confidence_score: Confidence score between 0 and 1 of the extracted
+            entity.
+        :ivar links: A collection of entity references in known data sources.
+        vartype links: list[~azure.ai.textanalytics.HealthcareEntityLink]
```

Comment:
:vartype links:

Reply:
I think I got all of these that were missing the ':'.